packages/flutter/lib/src/painting/box_decoration.dart

```diff
@@ -79,8 +80,15 @@ class BoxDecoration extends Decoration {
     this.borderRadius,
     this.boxShadow,
     this.gradient,
+    this.backgroundBlendMode,
     this.shape = BoxShape.rectangle,
-  }) : assert(shape != null);
+  }) : assert(shape != null),
+       // TODO(mattcarroll): Use "backgroundBlendMode == null" when Dart #31140 is in.
+       assert(
+         identical(backgroundBlendMode, null) || color != null || gradient != null,
+         'backgroundBlendMode applies to BoxDecoration\'s background color or'
+         'gradient, but no color or gradient were provided.'
```

Comment:
need a space after the `or` or you'll have the word `orgradient` in the assert message

Reply:
done